frontend/src/constants/paths.jsx

```diff
@@ -0,0 +1,21 @@
+const PATH_TO_HOMEPAGE = '/';
+const PATH_TO_CALENDAR = '/calendar';
+const PATH_TO_CREDITS = '/credits';
+const PATH_TO_DAILY_DRUGS = '/daily';
+const PATH_TO_LEXICON = '/lexicon';
+const PATH_TO_LOGIN = '/login';
+const PATH_TO_REGISTER = '/register';
+const PATH_TO_USERHUB = '/userhub';
+const PATH_TO_SETTINGS = '/settings';
+
+export {
+  PATH_TO_CALENDAR,
+  PATH_TO_CREDITS,
+  PATH_TO_DAILY_DRUGS,
+  PATH_TO_HOMEPAGE,
+  PATH_TO_LEXICON,
+  PATH_TO_LOGIN,
+  PATH_TO_REGISTER,
+  PATH_TO_SETTINGS,
+  PATH_TO_USERHUB,
+};
```

Comment:
How about having just export const instead of having this list?

Reply:
yup yup